include/libopencm3/stm32/g4/rcc.h

```diff
@@ -649,40 +696,53 @@
 #define RCC_BDCR_LSEDRV_SHIFT		3
 #define RCC_BDCR_LSEDRV_MASK		0x3
 
-#define RCC_BDCR_LSEBYP				(1 << 2)
-#define RCC_BDCR_LSERDY				(1 << 1)
-#define RCC_BDCR_LSEON				(1 << 0)
+#define RCC_BDCR_LSEBYP			(1 << 2)
+#define RCC_BDCR_LSERDY			(1 << 1)
+#define RCC_BDCR_LSEON			(1 << 0)
 /**@}*/
 
 /** @defgroup rcc_csr_values RCC_CSR - Control/Status register
  * @{
  */
-#define RCC_CSR_LPWRRSTF			(1 << 31)
-#define RCC_CSR_WWDGRSTF			(1 << 30)
-#define RCC_CSR_IWDGRSTF			(1 << 29)
-#define RCC_CSR_SFTRSTF				(1 << 28)
-#define RCC_CSR_BORRSTF				(1 << 27)
-#define RCC_CSR_PINRSTF				(1 << 26)
-#define RCC_CSR_OBLRSTF				(1 << 25)
-#define RCC_CSR_RMVF				(1 << 23)
+#define RCC_CSR_LPWRRSTF		(1 << 31)
+#define RCC_CSR_WWDGRSTF		(1 << 30)
+#define RCC_CSR_IWDGRSTF		(1 << 29)
+#define RCC_CSR_SFTRSTF			(1 << 28)
+#define RCC_CSR_BORRSTF			(1 << 27)
+#define RCC_CSR_PINRSTF			(1 << 26)
+#define RCC_CSR_OBLRSTF			(1 << 25)
+#define RCC_CSR_RMVF			(1 << 23)
 #define RCC_CSR_RESET_FLAGS	(RCC_CSR_LPWRRSTF | RCC_CSR_WWDGRSTF |\
 		RCC_CSR_IWDGRSTF | RCC_CSR_SFTRSTF | RCC_CSR_BORRSTF |\
 		RCC_CSR_PINRSTF | RCC_CSR_OBLRSTF)
 
-#define RCC_CSR_LSIRDY				(1 << 1)
-#define RCC_CSR_LSION				(1 << 0)
+#define RCC_CSR_LSIRDY			(1 << 1)
+#define RCC_CSR_LSION			(1 << 0)
 /**@}*/
 
 /** @defgroup rcc_crrcr RCC_CRRCR Clock Recovery RC register
  * @{
  */
-#define RCC_CRRCR_HSI48VAL_MASK		0x1ff
-#define RCC_CRRCR_HSI48VAL_SHIFT	7
+#define RCC_CRRCR_HSI48CAL_MASK		0x1ff
+#define RCC_CRRCR_HSI48CAL_SHIFT	7
 
-#define RCC_CRRCR_HSI48RDY	BIT1
-#define RCC_CRRCR_HSI48ON	BIT0
+#define RCC_CRRCR_HSI48RDY		(1 << 1)
+#define RCC_CRRCR_HSI48ON		(1 << 0)
 /**@}*/
```

Comment:
As far as I can tell, 99.9% of changes above this point are whitespace.  Please, just don't do that.  At the very least, don't do it in the same commit.

Reply:
This should be fixed now.

Reply:
much easier to read now! thanks!